### ISSUE TITLE
Option to print the compiler versions in a shell-friendly format ?

### ISSUE THREAD

**Issue by mmhat** [Kleidukos/get-tested#51]
Currently `get-tested` outputs JSON:
```
$ get-tested my-project.cabal
["9.8.2", "9.10.1"]
```
It was nice if there was an option to print that in a more shell-friendly format, i.e. one version per line, unquoted:
```
$ get-tested --format=shell my-project.cabal
9.8.2
9.10.1
```

Then one could do something like `for v in $(get-tested --shell); do cabal test all --with-compiler "ghc-${v}"; done`...

**Comment by Kleidukos** [comment]
Yes good idea, this would also lay the foundations for a `--format=nix` that was asked in #14.
